test/plot-schema.json

```diff
@@ -10278,14 +10278,15 @@
       "valType": "number"
      },
      "tickmode": {
-      "description": "Sets the tick mode for this axis. If *auto*, the number of ticks is set via `nticks`. If *linear*, the placement of the ticks is determined by a starting position `tick0` and a tick step `dtick` (*linear* is the default value if `tick0` and `dtick` are provided). If *array*, the placement of the ticks is set via `tickvals` and the tick text is `ticktext`. (*array* is the default value if `tickvals` is provided).",
+      "description": "Sets the tick mode for this axis. If *auto*, the number of ticks is set via `nticks`. If *linear*, the placement of the ticks is determined by a starting position `tick0` and a tick step `dtick` (*linear* is the default value if `tick0` and `dtick` are provided). If *array*, the placement of the ticks is set via `tickvals` and the tick text is `ticktext`. (*array* is the default value if `tickvals` is provided). If *sync*, the number of ticks will sync with the overlayed axis set by `overlaying` property.",
       "editType": "ticks",
       "impliedEdits": {},
       "valType": "enumerated",
       "values": [
        "auto",
        "linear",
-       "array"
+       "array",
+       "sync"
```

Comment:
Perhaps *sync* shouldn't be added to `minor` axes. 
If `major` ticks are *sync*, then *auto* option for `minor` ticks on that axis should copy minor ticks from the base axis. No?

Reply:
Yeah, removing.